pkg/controller/jenkins/configuration/user/seedjobs/seedjobs.go

```diff
@@ -42,6 +42,12 @@ const (
 	// AgentName is the name of seed job agent
 	AgentName = "seed-job-agent"
 
+	// AgentImage is the Image the agent will use
+	AgentImage = "jenkins/jnlp-slave:alpine"
+
+	// AgentServiceAccountName is the SA the agent will run as
+	AgentServiceAccountName = "default"
+
```

Comment:
Seed job agent definition would have to be refactored to use the new API and would be the default Agent in `[]Agents`

What do you think @tomaszsek?